camel/camel-mail/src/main/resources/project-defaults.yaml

```diff
@@ -0,0 +1,11 @@
+
+swarm:
+  mail:
+    smtp-servers:
+      greenmail:
+        host: localhost
+        port: 10110
+        username: user1@localhost
+        password: password
+  deployment:
+    com.icegreen:greenmail-webapp.war:
```

Comment:
Wow, this is cool!

Did you maybe want to add the version number, like it was there in `main`? Or it's enough to depend on the right version in the POM, I guess?